jerry-core/ecma/base/ecma-globals.h

```diff
@@ -1825,12 +1825,23 @@ typedef struct
 /**
  * Function callback descriptor of a %TypedArray% object getter
  */
-typedef ecma_number_t (*ecma_typedarray_getter_fn_t) (lit_utf8_byte_t *src);
+typedef ecma_value_t (*ecma_typedarray_getter_fn_t) (lit_utf8_byte_t *src);
 
 /**
  * Function callback descriptor of a %TypedArray% object setter
  */
-typedef void (*ecma_typedarray_setter_fn_t) (lit_utf8_byte_t *src, ecma_number_t value);
+typedef ecma_value_t (*ecma_typedarray_setter_fn_t) (lit_utf8_byte_t *src, ecma_value_t value);
+
+#if ENABLED (JERRY_BUILTIN_BIGINT)
+/**
+ * Checks whether a given typedarray is BigInt type or not.
+ *
+ * Note: this macro should be updated when new types are added
+ **/
+#define ECMA_TYPEDARRAY_IS_BIGINT_TYPE(id) \
```

Comment:
Probably I wasn't clear. This macro was in the right place, please put it back. Regarding the comment after "Note:" should go here:

```
  ECMA_FLOAT32_ARRAY,       /**< Float32Array */
  ECMA_FLOAT64_ARRAY,       /**< Float64Array */
  <- the comment after "Note:" should go here, 
     since if somebody changes these types, they see that 
     ECMA_TYPEDARRAY_IS_BIGINT_TYPE should be updated as well
  ECMA_BIGINT64_ARRAY,      /**< BigInt64Array */
  ECMA_BIGUINT64_ARRAY,     /**< BigUInt64Array */
} ecma_typedarray_type_t;
```

Reply:
An example here:
https://github.com/jerryscript-project/jerryscript/blob/master/jerry-core/parser/js/js-scanner.h#L139